megamek/src/megamek/server/Server.java

```diff
@@ -33784,6 +32561,8 @@ public boolean checkForCollapse(Building bldg,
                 recheckLoop = false;
                 // recheck if the basement collapsed since the basement falls
                 // might trigger a greater collapse.
+
+                // TODO : Windchild the recheck loop in this region all register as issues on my IDE
```

Comment:
The recheck loop here all register as unused by my IDE. Is that a bug or just old code?

Reply:
The two if statements that follow the for loop should be inside it.